platform/barefoot/bfn-platform.mk

```diff
@@ -1,5 +1,13 @@
 BFN_PLATFORM = bfnplatform_1.0.0_amd64.deb
-$(BFN_PLATFORM)_URL =	"https://github.com/barefootnetworks/sonic-release-pkgs/raw/sde-sai1.3.3/bfnplatform_1.0.0_amd64.deb"
 
+ifeq ($(CONFIGURED_P4_STACK),p4_14)
```

Comment:
if we should have it -we need to modify base (slave.mk) makefile with barefoot specific variables. Probably we could try, but in current variant it would be easy to propose to Azure.